platform_versioned_docs/version-23.4.0/functionality_matrix/functionality_matrix.mdx

```diff
@@ -22,11 +23,17 @@ nf-launcher versions prefixed with j17 refer to Java version 17; j11 refers to J
 | 23.3.0           | j17-23.04.3         | 23.04.3          | 2.1            |
 | 23.2.0           | j17.23.04.2-up3     | 23.04.2          | 2.1            |
 | 23.1.3           | j17-23.04.1         | 23.04.1          | 2.1            |
-| 22.4.1           | j17-22.10.6         | 22.10.6          | -              |
-| 22.4.0           | j17-22.10.6         | 22.10.6          | -              |
+
+:::note 
+Nextflow edge versions later than 24.04.2 use Fusion 2.4 by default. 
+:::
+
+(* - Nextflow version 24.04.2 will be the baseline version of an upcoming Platform release, but you can take advantage of [new Nextflow features](https://nextflow.io/blog/2024/nextflow-2404-highlights.html) in current Platform versions by specifying `NXF_VER=24.04.2` in your configuration.)
```

Comment:
Per Google Developer Style, it is recommended to avoid hinting at future state. Maybe 24.04.17 becomes the baseline? I don't know that we've actually adopted this prohibition, though, but wanted to mention.

Reply:
That's a good point, especially as it sets an expectation that may be delayed or rolled back etc.

Reply:
Agree on removing this - it also seems to essentially duplicate the bit below about how to change versions.